### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by jesusHCG**
### Description
Corrections in the test snapshop

### Related GitHub issue or JIRA story
https://jira.autodesk.com/browse/DYN-2072
https://jira.autodesk.com/browse/DYN-2116

### Checklist
- [ ] PR title reflects the changes it introduces. Consider using this template: QNTM-XXXX: Implement X Feature
- [ - ] Code compiles correctly
- [ - ] No linting errors
- [ - ] All tests passing
- [ - ] Updated README.md / documentation (if necessary)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ Reusable lightweight library component based on React.js
 ## Set up
 Installing all dependencies
 
+    $ npm install typescript@3.5.3 --save-dev
```
Comment (mjkkirschner) [inline README.md]: doesn't this get done as part of the below package.json?

File: __tests__/Snapshottest/__snapshots__/UIOutputComparisonTests.tsx.snap [hunk 2]
```diff
@@ -149,8 +149,8 @@ exports[`LibraryContainer Test UI rendering of Library Item and child components
     <div
       className="LibraryItemHeader"
       onClick={[Function]}
+      onMouseEnter={[Function]}
```
Comment (mjkkirschner) [inline __tests__/Snapshottest/__snapshots__/UIOutputComparisonTests.tsx.snap]: looks good.

File: .gitignore [hunk 3]
```diff
@@ -38,3 +38,4 @@ dist
 
 # Optional REPL history
 .node_repl_history
+package-lock.json
```
Comment (mjkkirschner) [inline .gitignore]: should we consider committing this?

File: README.md (hunk 1 above)
Comment (jesusHCG) [reply README.md]: I have done several tests and I have recently downloaded version 3.6.2 which generates the following error in the build:
https://github.com/microsoft/TypeScript/issues/32333
I found that forcing version 3.5.3 this error does not occur.

File: .gitignore (hunk 3 above)
Comment (jesusHCG) [reply .gitignore]: It is not necessary but in the end it is a file that we should not upload, I saw it convenient to add it to git ignore, but i can remove this from the file.

**Comment by mjkkirschner** [comment]
Then you should modify the package.json to force that version by removing the ^ symbol from typescript rather than forcing it here

File: README.md (hunk 1 above)
Comment (jesusHCG) [reply README.md]: Thanks for the recommendation, I already made the change

**Comment by jesusHCG** [comment]
Thanks for the recommendation, I already made the change

**Comment by mjkkirschner** [comment]
LGTM

**Pull request merged**